### DISCUSSION TITLE
Any idea of how to use this with Android

### DISCUSSION THREAD

**Discussion by czadikem** [filips123/PWAsForFirefox#87] [Q&A] [answered]
Any idea of how I can use this with Android?  If not then what can I use instead.  I don't want to use the WebApps app or anything chrome based.

**Comment by filips123** [comment, marked as answer]
You can use normal Firefox for Android which already supports PWAs. Just click on three dots to open menu and then "Install" or "Add to home screen".

However, If the website doesn't provide PWA manifest, you will just add an icon to your home screen that will open the browser and sti display address bar and other browser UI.

**Reply by Kuuh4** [reply]
Hi! I'd like to know if it's possible to inject a pseudo-manifest in order to be able to save the website as PWA, or if more steps/structure would be needed.
For some reason my homeserver (CasaOS) that consists of a singular webpage as interface is able to be installed as PWA, so I thought that it'd be possible to trick the browser to save other pages as well.

**Reply by filips123** [reply]
For Android? Well, I don't know, this project is only for desktop Firefox...

Firefox for Android wants the website to have web app manifest set up, and CasaOS probably has it, so that's why it works. But for websites without the manifest, I don't know if it's possible to trick Firefox.

**Reply by Kuuh4** [reply]
Thank you so much for your time :3
